ovs-service/src/main/java/org/dcsa/ovs/controller/OVSRestController.java

```diff
@@ -20,6 +20,8 @@
 @RequiredArgsConstructor
 public class OVSRestController {
```

Comment:
We never used the term Rest in our controller. Propose to rename to ServiceScheduleController to align with OAS